components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -94,22 +95,41 @@ public void addPolicy(String policyLevel, Policy policy) throws APIMgtDAOExcepti
             connection = DAOUtil.getConnection();
 
             //TODO : instead of checking policyLevel, check class type, and remove passing policy level to here
```

Comment:
Shall we add all the missing method comments as in https://github.com/wso2/carbon-apimgt/blob/C5/components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/LabelDAOImpl.java#L32